### PR TITLE
Rename deployment policy to runtime policy

### PR DESCRIPTION
**Pull request by BrendanSchell**
## Description
Renames deployment policy to runtime policy.

Closes https://github.com/OpenMined/Heartbeat/issues/1554.

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
Tested by running through enclave notebooks after rename.

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

**Review by rasswanth-s — APPROVED** [review]
LGTM.

Let also wait for feedback from @madhavajay  before merge
https://miro.com/app/board/uXjVNqD_gj0=/?openComment=3458764593016520472

**Review by itstauq — APPROVED** [review]

**Pull request merged**
